### DISCUSSION TITLE
Advanced Field Help

### DISCUSSION THREAD

**Discussion by Fluzzroz** [burkybang/SuiteAdvanced-Feedback#78] [Ideas]
SuiteAdvanced has implemented a nice feature that lets you see more stuff about a field when the user clicks on the field name (ex: link towards a custom field definition and table with possible values of a list field).

This is really cool.

However, someone already does it better : https://chrome.google.com/webstore/detail/netsuite-advanced-field-h/ehgcekpdldhgfjpionbkilpnaahjiejj

I would suggest one of the following:
1. remove that feature from SuiteAdvanced
2. have a setting to turn it off
3. implement it fully. Here's what the other guy does better: the table is easier to read with alternate background color, sublist column labels are now clickable like body fields and give you the same info, the user can set a new value regardless of the display type

**Comment by burkybang** [comment]
Excellent constructive feedback. Thank you very much. I have not used that extension before, so it's helpful to have that comparison. I will absolutely consider your suggestions.

**Comment by burkybang** [comment]
New discussion: https://github.com/burkybang/SuiteAdvanced-Feedback/discussions/92
